code/LINDA/LINDA_fire.dm

```diff
@@ -26,13 +26,23 @@
 	if((exposed_temperature > PLASMA_MINIMUM_BURN_TEMPERATURE) && air_contents.toxins > 0.5)
 		igniting = 1
 
+	var/obj/effect/decal/cleanable/liquid_fuel/liquid = locate() in src
```

Comment:
Umm this isn't LINDA code, it's turf code. Since hotspot_expose is a turf proc, there's nowhere else to put this in.

Your recommendation most likely wouldn't give the desired result as the plasma would try to spread, no?